CodeReview/src/overlays/CodeReviewCommentOverlay.cpp

```diff
@@ -43,6 +43,10 @@ CodeReviewCommentOverlay::CodeReviewCommentOverlay(Visualization::Item* associat
 	Super{{associatedItem}, style}
 {
 	setAcceptedMouseButtons(Qt::AllButtons);
+	// TODO what happens now in initializeForms() because of the item() function which takes commentInput_ as item storage?
```

Comment:
The `initializeForms` just describes what an "update" to the item would look like, but since you're not really doing anything special to it, `commentInput_` will be basically left alone :)

Please remove this TODO.